lib/cmd/fh3/projects/create.js

```diff
@@ -0,0 +1,61 @@
+/* globals i18n */
+var common = require("../../../common");
+var ini = require('../../../utils/ini');
+var templates = require('../../common/templates.js');
+var util = require('util');
+
+
+module.exports = {
+  'desc' : i18n._('Create project'),
+  'examples' :
+  [{
+    cmd : 'fhc projects create --title=<title>',
```

Comment:
Little bit misunderstanding:
Old command format was: `fhc projects create <project-title>`
Currently: `fhc projects create --title=<title>`

While I agree to keep it consistent let's not break previous implementations.
Am I getting it right?